File: server-scala-play-server/app/model/ConsumerInviteResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ConsumerInviteResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class ConsumerInviteResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  inviterId: Option[Long],
  inviter: Option[AccountShortResponse],
  connection: Option[ConnectionResponse],
  inviteText: Option[String],
  token: Option[String],
  albumCoverUrl: Option[String],
  albumTitle: Option[String],
  appName: Option[String],
  welcomeText: Option[String],
  application: Option[ApplicationShortResponse],
  retailerLocation: Option[RetailerLocationResponse],
  offer: Option[OfferShortResponse],
  album: Option[AlbumResponse],
  returning: Option[String]
)

object ConsumerInviteResponse {
  implicit lazy val consumerInviteResponseJsonFormat: Format[ConsumerInviteResponse] = Json.format[ConsumerInviteResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/ConsumerInviteResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class ConsumerInviteResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public long InviterId { get; set; }
    public AccountShortResponse Inviter { get; set; }
    public ConnectionResponse Connection { get; set; }
    public string InviteText { get; set; }
    public string Token { get; set; }
    public string AlbumCoverUrl { get; set; }
    public string AlbumTitle { get; set; }
    public string AppName { get; set; }
    public string WelcomeText { get; set; }
    public ApplicationShortResponse Application { get; set; }
    public RetailerLocationResponse RetailerLocation { get; set; }
    public OfferShortResponse Offer { get; set; }
    public AlbumResponse Album { get; set; }
    public string Returning { get; set; }
}


